app/paths.ts

```diff
@@ -55,7 +55,8 @@ const paths = {
     accessNeedsAdditionalLanguage: '/appointment-add-language-details',
     otherNeeds: 'TODO',
     datesToAvoid: 'TODO',
-    checkAndSend: '/cma-requirements/check-your-answers'
+    checkAndSend: '/cma-requirements/check-your-answers',
+    whatToExpect: '/expect-case-management-appointment'
```

Comment:
guidance pages should be moved to common as they do not provide any information about the case or state and should be publically available